src/utils/prim-lexing.fs

```diff
@@ -2,6 +2,31 @@
 
 #nowarn "47" // recursive initialization of LexBuffer
 
+namespace Microsoft.FSharp.Compiler
+
+type ITextLine =
```

Comment:
`TextLine` in Roslyn is a struct, I'm not sure making this an interface makes sense. Is there a way to also make our own struct for it?